x-pack/libbeat/docs/aws-credentials-config.asciidoc

```diff
@@ -0,0 +1,80 @@
+[float]
+=== AWS Credentials Configuration
+To configure AWS credentials, there are two different ways supported:
+
+[float]
+==== Supported Formats
+* Use `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and/or `AWS_SESSION_TOKEN`
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: aws
+  period: 300s
+  metricsets:
+    - ec2
+  access_key_id: '${AWS_ACCESS_KEY_ID:""}'
```

Comment:
If I understand this correctly, the other example (default credentials) would already use these env vars?

If that's the case it may make sense to leave this format for manually introduced credentails. So something like `access_key_id: '<your_access_key>'` may be easier to understand?'

Reply:
Also after this change you can probably revisit `aws.yml`

Reply:
In this example, I want to show you can either give the access key id directly `access_key_id: 'your_access_key'` or use environment variable for it: `access_key_id: '${AWS_ACCESS_KEY_ID:""}'`. Regarding to `aws.yml`, I'm leaning towards removing the aws credential part from all metricset configuration examples:
```
access_key_id: '${AWS_ACCESS_KEY_ID:""}'
secret_access_key: '${AWS_SECRET_ACCESS_KEY:""}'
session_token: '${AWS_SESSION_TOKEN:""}'
default_region: '${AWS_REGION:us-west-1}'
```
WDYT?

Reply:
SGTM, instead of removing showing a them commented out and without the env var would probably be helpful

Reply:
Sounds good! Done. Thanks!